modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2008,33 +1999,15 @@ def last_valid_index_builder(df):
 
     # END Column/Row partitions reduce operations
 
-    def describe(
-        self,
-        percentiles: np.ndarray,
-        datetime_is_numeric: bool,
-    ):
+    def describe(self, percentiles: np.ndarray):
         # Use pandas to calculate the correct columns
         empty_df = (
             pandas.DataFrame(columns=self.columns)
             .astype(self.dtypes)
-            .describe(
-                percentiles, datetime_is_numeric=datetime_is_numeric, include="all"
-            )
+            .describe(percentiles, include="all")
         )
         new_index = empty_df.index
 
-        # Note: `describe` convert timestamp type to object type
-        # which results in the loss of two values in index: `first` and `last`
-        # for empty DataFrame.
-        if not any(map(is_numeric_dtype, empty_df.dtypes)) and not datetime_is_numeric:
```

Comment:
"Removed argument datetime_is_numeric from [DataFrame.describe()](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.describe.html#pandas.DataFrame.describe) and [Series.describe()](https://pandas.pydata.org/docs/reference/api/pandas.Series.describe.html#pandas.Series.describe) as datetime data will always be summarized as numeric data ([GH34798](https://github.com/pandas-dev/pandas/issues/34798))"

In pandas 2.0 this code branch is not reachable.